programs/token-metadata/program/src/processor/delegate/delegate.rs

```diff
@@ -48,6 +49,9 @@ impl Display for DelegateScenario {
                     "ProgrammableConfigItem".to_string()
                 }
             },
+            Self::Holder(role) => match role {
+                HolderDelegateRole::PrintDelegate => "PrintDelegate".to_string(),
+            },
```

Comment:
Out of scope for this PR, but I think these display trait implementations are strange in that here it implements one set of strings in Pascal case, but elsewhere the roles have their own display traits with snake case.  I think some of them were used for a default RuleSet we made that used delegate scenarios, so maybe no changes needed ever, but just wanted to call out that it seems kind of inconsistent.

Reply:
Discussed offline